src-java/northbound-service/northbound/src/main/java/org/openkilda/northbound/converter/FlowMapper.java

```diff
@@ -149,9 +150,43 @@ public FlowResponseV2 toFlowResponseV2(FlowDto flowDto) {
     @Mapping(target = "type", ignore = true)
     public abstract FlowRequest toFlowRequest(FlowRequestV2 request);
 
+    @Mapping(target = "flowId", source = "id")
+    @Mapping(target = "bandwidth", source = "maximumBandwidth")
+    @Mapping(target = "transitEncapsulationId", ignore = true)
+    @Mapping(target = "type", ignore = true)
+    @Mapping(target = "detectConnectedDevices", ignore = true)
+    public abstract FlowRequest toFlowRequest(FlowCreatePayload request);
+
     @Mapping(target = "outerVlanId", source = "vlanId")
     public abstract FlowEndpoint mapFlowEndpoint(FlowEndpointV2 input);
 
+    @Mapping(target = "switchId", source = "datapath")
+    @Mapping(target = "outerVlanId", source = "vlanId")
+    @Mapping(target = "trackLldpConnectedDevices", source = "detectConnectedDevices.lldp")
+    @Mapping(target = "trackArpConnectedDevices", source = "detectConnectedDevices.arp")
+    public abstract FlowEndpoint mapFlowEndpoint(FlowEndpointPayload input);
```

Comment:
should we also map connected devices fields in method
 `public abstract FlowEndpoint mapFlowEndpoint(FlowEndpointV2 input);` ?